web-client/src/modules/requests/routes.tsx

```diff
@@ -26,23 +27,19 @@ const TabbedPostsPage = lazy(() => import('./pages/TabbedPostsPage'));
 const Routes = (): ReactElement => (
   <Suspense fallback={<LoadingWrapper />}>
     <Switch>
-      <Route
-        path={AcceptedRequestsLocation.path}
-        component={AcceptedRequestsContainer}
-        exact
-      />
       <Route
         path={FindRequestsLocation.path}
         component={FindRequestsContainer}
-        exact
       />
-      <Route
+      <ProtectedRoute
+        path={AcceptedRequestsLocation.path}
+        component={AcceptedRequestsContainer}
+      />
+      <ProtectedRoute
         path={OpenRequestsLocation.path}
         component={OpenRequestsContainer}
-        exact
       />
-      <Route path={ListMyPostsLocation.path} component={TabbedPostsPage} />
-
+      <ProtectedRoute path={MyPostsLocation.path} component={TabbedPostsPage} />
```

Comment:
I wouldn't use protected route here, and would rather have it be controlled by the masterpage.

Reply:
let's speak more about this before making this change so for now, I am reverting it